enterprise/server/usage/usage.go

```diff
@@ -37,6 +56,19 @@ const (
 	redisCountsKeyPrefix = redisUsageKeyPrefix + "counts/"
 
 	redisTimeKeyFormat = time.RFC3339
+
+	redisUsageLockKey    = "lock.usage"
+	redisUsageLockExpiry = 45 * time.Second
```

Comment:
Can you please comment these (and give an example of time.RFC3339? It is not an RFC i know by heart, unlike RFC 1149)